fly.toml

```diff
@@ -11,33 +9,8 @@ processes = []
 [env]
   PORT = "8080"
 
-[experimental]
-  allowed_public_ports = []
-  auto_rollback = true
```

Comment:
This is not neccesary for Fly.io v2

Reply:
I'm guessing for the same reason, that it's configured already by default, like mentioned here:  https://github.com/upleveled/security-vulnerability-examples-next-js-postgres/pull/85/files#r1190099894